service/src/main/java/bio/terra/profile/service/crl/CrlService.java

```diff
@@ -0,0 +1,70 @@
+package bio.terra.profile.service.crl;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.cloudres.google.billing.CloudBillingClientCow;
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.profile.app.configuration.AzureConfiguration;
+import bio.terra.profile.service.crl.exception.CrlInternalException;
+import bio.terra.profile.service.crl.exception.CrlSecurityException;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.resourcemanager.resources.ResourceManager;
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.io.IOException;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CrlService {
```

Comment:
This repo uses CRL for cloud access, but doesn't actually create any cloud resources, so no need for Janitor.